pyproject.toml

```diff
@@ -28,7 +28,7 @@ rstr = "^3.2.2"
 typing-extensions = "^4.8.0"
 python-dateutil = "^2.8.2"
 tiktoken = ">=0.5.1"
-nltk = "^3.8.1"
```

Comment:
Do we really need to fix the version of NLTK?  Can we cap it at a bigger range to make it more compatible with people who are integrating it?

Reply:
Agreed, probably need something like `nltk = ">3.0,<=3.8.1`

Reply:
cap is reasonable. surprised we dont have more issues with the ^ in here

Reply:
can cap it.